core/drivers/crypto/crypto_api/include/drvcrypt.h

```diff
@@ -50,6 +50,7 @@ struct drvcrypt_buf {
 enum drvcrypt_algo_id {
 	CRYPTO_HASH = 0, /* Hash driver */
 	CRYPTO_HMAC,	 /* HMAC driver */
+	CRYPTO_CMAC,	 /* CIPHER MAC driver */
```

Comment:
s/CIPHER MAC/Cipher MAC/   ?
or simply CMAC ?

Reply:
That will be ```CMAC driver```